src/Evolution/Systems/ScalarAdvection/VelocityField.hpp

```diff
@@ -0,0 +1,24 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/ScalarAdvection/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace ScalarAdvection::Tags {
+/*!
+ * \brief Compute the advection velocity field of ScalarAdvection system
+ */
+template <size_t Dim>
+struct VelocityFieldCompute {
```

Comment:
This tag doesn't  work. It's missing the inheritance and `argument_tags`, as well as a base simple tag, `return_type`, and possibly more

Reply:
Yep, this part was subtle and I should've asked you. My first try, actually, was doing exactly as same as `Characteristics.hpp` : 
```
struct VelocityFieldCompute : VelocityField, db::ComputeTag {
```
but then I was stuck how to deal with `argument_tags` here, since `VelocityFieldCompute::function` requires inertial coordinates as an argument. To me it seemed little bit awkward to declare inertial coordinates as one of `Tags`..?

Reply:
It's not ideal in general for compute tags to take the coordinates as arguments, but we currently don't have a tested way of avoiding it in all cases. That's on the "TODO" list but hasn't reached the top yet. We have several tags that take the inertial coordinates as their argument. One example is the constraint damping functions for the GH system. https://github.com/sxs-collaboration/spectre/blob/develop/src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/ConstraintGammas.hpp as an example

Reply:
Ah, okay! Now I see how to put coordinates as tags.